.gitlab-ci.yml

```
@@ -7,8 +7,9 @@ before_script:
     - export OMP_NUM_THREADS=1
     - export CMAKE_VERSION=3.10.3
     - export CTEST_OUTPUT_ON_FAILURE=1
+    - export J=$(nproc) && echo ${J}
```

```suggestion
    - export J=4 && echo Using ${J} cores during build
```

The CI machine is rather dated (Westmere Xeons E5540 with 8 cores and 16 threads total and a single spinning hdd) and the runner are shared across multiple projects (and can run in parallel). Thus I'd prefer to fix `J` to 4 so that the system doesn't freeze entirely during peaks.